tests/torch/speculative/test_suffix_automaton.py

```
@@ -570,6 +570,257 @@ def test_extend_ngram_cuda_graph(self):
+        max_draft_len = 4
+        manager.prepare(request_ids, max_draft_len)
+
+        # Token 99 doesn't exist in any SA
+        accepted_tokens = torch.tensor(
+            [[99, 0, 0, 0, 0], [99, 0, 0, 0, 0]],
+            dtype=torch.int32,
+            device="cuda",
+        )
+        num_accepted_tokens = torch.tensor([1, 1], dtype=torch.int32, device="cuda")
+
+        match_len, draft_tokens = manager.extend_global(
```

Ruff (0.15.5)

[warning] Unpacked variable `draft_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)